src/kemal/helpers/helpers.cr

```diff
@@ -149,14 +149,14 @@ private def multipart(file, env : HTTP::Server::Context)
   end
 
   if endb == 0
-    endb = fileb
+    endb = fileb - 1
   end
 
   if startb < endb && endb <= fileb
     env.response.status_code = 206
     env.response.content_length = endb - startb
```

Comment:
@crisward This should be `(endb - startb) + 1`. 
e.g. if the range is `Range: bytes=0-1023` the content length would be 1023 but it should be 1024.

Reply:
Well spotted. 157 should be `endb - startb + 1`

Reply:
Something really nasty about code with lots of +1 / -1 's everywhere...

Reply:
On Line 155 it should be `if startb < endb && endb < fileb`.